### ISSUE TITLE
Extra encoder layer by mistake?

### ISSUE THREAD

**Issue by filipinascimento** [jianz94/e-lstm-d#2]
Hi, awesome work. Very interesting methods.

I looked at the code and found something that I'm sure if it was intended or if it is a mistake. The model uses an extra layer even if you send just one layer for the encoder. Looking at the source code of `model.py` at line 60 I believe there should be a if, so it will create the extra layers without the input argument. Currently if you pass one layer it is going to always create a duplicate of the first layer.

**Comment by jianz94** [comment]
Yes, it is a mistake. There should be an else before line 60. Thanks for your comment.
